<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="RFID_SYSTEM_LABEL">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:22pt; font-weight:600;&quot;&gt;RFID SYSTEM&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="RFID_CONNECTED">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RFID connected</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="RFID_Box">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>120</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>90</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SYSTEM STATUS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ADD_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>ADD</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CREATING MENU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DELETE_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>300</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CONFIGURE_Button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>CONFIGURE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SHOW_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>380</y>
      <width>80</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string>SHOW</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DEVICE MENAGER</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ADD_Device">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>ADD DEVICE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DELETE_Device">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>DELETE DEVICE</string>
    </property>
   </widget>
   <widget class="QLabel" name="SQL_Label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SQL connected</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="SQL_Box">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>72</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>CheckBox</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Name">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LastName">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>260</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Name_Label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>230</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="LastName_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>230</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LastName</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RFID">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Name_Label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>RFID</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Device_Box">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SEARCH_Device">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>SEARCH DEVICE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CONNECT">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>CONNECT</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DISCONNECT">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>180</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>DISCONNECT</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuKonfiguracja">
    <property name="title">
     <string>Konfiguracja </string>
    </property>
    <addaction name="actionDodaj_Urz_dzenie"/>
   </widget>
   <addaction name="menuKonfiguracja"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDodaj_Urz_dzenie">
   <property name="text">
    <string>Dodaj UrzÄ…dzenie</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
